__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/categories", () => {
+  describe("GET", () => {
+    test("200: responds with an array of category objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.categories)).toBe(true);
+          expect(body.categories.length > 0).toBe(true);
+          body.categories.forEach((category) => {
```

Comment:
Since you have the test data, you can explicitly say what length you expect categories to be!